--- gringo-3.0.5-source/libgringo/src/rule.cpp	2013-03-08 14:45:28.000000000 +0100
+++ gringo-3.0.5-source-fasp/libgringo/src/rule.cpp	2013-04-04 21:33:24.018787078 +0200
@@ -15,6 +15,9 @@
 // You should have received a copy of the GNU General Public License
 // along with gringo.  If not, see <http://www.gnu.org/licenses/>.
 
+#include <iostream>
+using namespace std;
+
 #include <gringo/rule.h>
 #include <gringo/lit.h>
 #include <gringo/instantiator.h>
@@ -29,6 +32,7 @@
 	, head_(head)
 	, body_(body.release())
 	, grounded_(false)
+	, constant_(1.0)
 {
 	if(head) head->head(true);
 }
